kitty/kitty.go

```diff
@@ -28,6 +30,17 @@ func (e *Encoder) Encode(img image.Image) error {
 	if width == 0 || height == 0 {
 		return nil
 	}
+	max_size := 10000
+	if width > max_size || height > max_size {
+		if width > height {
+			height = height * max_size / width
```

Comment:
`height *=`  and `width *=` ?

Reply:
I intentionally multiplied first and then divided to avoid rounding due to integer division.
However, I found that imaging.Resize (https://github.com/disintegration/imaging)
preserves the aspect ratio if width or height is 0. So the result was same.

I'd like to change them by simply substituting 0 without doing complicated things.